stubs/c-libressl/run.c

```diff
@@ -87,7 +88,7 @@ int main(int argc, char **argv) {
                 }
 
                 /* Filter away non-ASCII charaters */
```

Comment:
Nit: This comment is now somewhat misleading.

(There's also a typo, missing character "c" in "characters" 😄)